### PR TITLE
Added glyph of stars decorations and death check for cerus achievement

### PR DESCRIPTION
**Pull request by Linkaaaaa**

### CODE REVIEW

File: GW2EIEvtcParser/EIData/ProfHelpers/Ranger/DruidHelper.cs [hunk 1]
```diff
@@ -58,5 +59,36 @@ public static bool IsCelestialAvatarTransform(long id)
...
+                    var connector = new PositionConnector(effect.Position);
+                    replay.Decorations.Add(new CircleDecoration(360, lifespan, color, 0.5, connector).UsingFilled(false).UsingSkillMode(skill));
+                    replay.Decorations.Add(new IconDecoration(ParserIcons.EffectGlyphOfTheStars, CombatReplaySkillDefaultSizeInPixel, CombatReplaySkillDefaultSizeInWorld, 0.5f, lifespan, connector).UsingSkillMode(skill));
+                }
+            }
+
+            // Glyph of the Stars (Celestial Avatar)
+            if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.DruidGlyphOfTheStarsCA, out IReadOnlyList<EffectEvent> glyphOfTheStarsCA))
+            {
+                var skill = new SkillModeDescriptor(player, Spec.Druid, GlyphOfTheStarsCA, SkillModeCategory.Heal);
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/ProfHelpers/Ranger/DruidHelper.cs:83]: Should also be in Important Buffs (Stab)

**Comment by Linkaaaaa** [comment]
Speaking of skill categories, should we have one for CC ? for example right now gravity well on chrono and vital draw on harbinger have the default not applicable category

**Comment by EliphasNUIT** [comment]
If it is not enabled on CR by default, why not.

File: GW2EIBuilders/Resources/JS/global.js [hunk 2]
```diff
@@ -185,13 +185,20 @@ const SkillDecorationCategory = {
...
+    "CC": 1 << 7,
 };
 
 let AllSkillDecorations = 0;
 for (let key in SkillDecorationCategory) {
     AllSkillDecorations |= SkillDecorationCategory[key];
 }
-let DefaultSkillDecorations = AllSkillDecorations;
+
+let DefaultSkillDecorations = 0;
```
Comment (EliphasNUIT) [inline GW2EIBuilders/Resources/JS/global.js:196]: let DefaultSkillDecorations = AllSkillDecorations & ~SkillDecorationCategory.CC;

File: GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/NecromancerHelper.cs [hunk 3]
```diff
@@ -314,7 +314,7 @@ internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player,
             // Reaper's Mark (Staff 5)
             if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.NecromancerReapersMark, out IReadOnlyList<EffectEvent> reapersMarks))
             {
-                var skill = new SkillModeDescriptor(player, Spec.Necromancer, ReapersMark);
+                var skill = new SkillModeDescriptor(player, Spec.Necromancer, ReapersMark, SkillModeCategory.CC);
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/NecromancerHelper.cs:317]: Is the category needed for the non activated version? For example we don't add "Portal" category to non opened portal skills.

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**